README.md

```diff
@@ -30,6 +30,7 @@
 - [rahuldkjain/GitHub Profile Readme Generator](https://github.com/rahuldkjain/github-profile-readme-generator#readme) - 🚀 Generate github profile readme easily with the latest add-ons like visitors count, github stats, etc using minimal UI.
 - [GitHub Profilinator](https://github.com/rishavanand/github-profilinator#readme) - 🚀 This tool contains mini GUI components that you can hook together to automatically generate markdown code for a perfect readme.
 - [REHeader](https://github.com/khalby786/REHeader#readme) - Generate beautiful header images for your github profile readmes.
+- [github-profile-readme-maker](https://github.com/VishwaGauravIn/github-profile-readme-maker) - Create your perfect GitHub Profile ReadMe in the best possible way.
```

Comment:
According to alphabetical order this one should come before "GitHub Profilinator". 
Also update the name to "GitHub Profile Readme Maker".

Reply:
sure!

Reply:
one more question sir , as I'm new in this..

should I make new branch whenever I am making a new pull request to add feature or should continue with existing branch like this dev1 ? 
is it okay to have as many as branches in a repository ?

Reply:
You should create one branch per feature/bugfix/etc.., by this way it will be cleaner & also makes PR reviews easier Also once the branch is merged to main branch you should delete the branch.

For example now once your `dev1` branch gets merged you should delete it from your cloned repo and create a new one if you want to add any new feature.

Note: Also try naming your branches meaningfully.